src/Compilers/CSharp/Test/Emit2/Semantics/ExperimentalAttributeTests.cs

```diff
@@ -0,0 +1,910 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests;
+
+// These tests cover the handling of System.Diagnostics.CodeAnalysis.ExperimentalAttribute.
+public class ExperimentalAttributeTests : CSharpTestBase
+{
+    private const string experimentalAttributeSrc = """
+#nullable enable
+
+namespace System.Diagnostics.CodeAnalysis
+{
+    [AttributeUsage(AttributeTargets.All, Inherited = false)]
+    public sealed class ExperimentalAttribute : Attribute
+    {
+        public ExperimentalAttribute(string diagnosticId) { }
+
+        public string? UrlFormat { get; set; }
+    }
+}
+""";
+
+    private const string DefaultHelpLinkUri = "https://msdn.microsoft.com/query/roslyn.query?appId=roslyn&k=k(CS8305)";
```

Comment:
>https://msdn.microsoft.com/query/roslyn.query?appId=roslyn&k=k(CS8305) [](http://example.com/codeflow?start=47&length=70)

The link reports this error is unrecognized currently. Is that expected?

Reply:
Does the spec state this is the default URI, when not set?

Reply:
That's the link for existing error code `WRN_Experimental` used for Windows Experimental attribute. 
@BillWagner, should we file an issue to create this page?